### PR TITLE
 Finished first draft of pipelines list page.

### PR DESCRIPTION
**Pull request by ewels**
First draft of the new pipelines page, using the GitHub API to fetch the projects.

Projects are split into two lists according to whether they have any releases or not. In a future update I can add a third list for repositories that are archived.

The styling isn't super pretty, but hopefully it's a start. We can also think about what other information would be useful to display.

---

Screenshot:
image

---

Demo with a different GitHub organisation that has more repos (with releases):

127 0 0 1_4000_pipelines

### CODE REVIEW

**Comment by ewels** [comment]
I added a third section for archived pipelines. Looks like this:

image

**Comment by ewels** [comment]
Appended a little extra documentation to correspond to nf-core/tools#17

File: docs.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ The quickest place to get help is on the nf-core Gitter channel: [https://gitter
 For now, the guidelines are pretty simple. The below gives an outline of what is required.
 For more detail, please see the [**list of lint test error codes**](errors).
 
+If in doubt, we recommend using the [nf-core/cookiecutter](https://github.com/nf-core/cookiecutter)
+pipeline template to get started. This generates a skeleton pipeline with all of the featuers
```
Comment (apeltzer) [inline docs.md]: featuers -> features

**Review by apeltzer — CHANGES_REQUESTED** [review]
Small typo, but everything else looks good to me :-) checked in the train to utilize my time more efficiently

**Comment by ewels** [comment]
Good spot, thanks! Fixed 👍

**Review by apeltzer — APPROVED** [review]

**Comment by ewels** [comment]
https://nf-core.github.io/pipelines 🎉

**Comment by apeltzer** [comment]
Love it :champagne:

**Pull request merged**